app/api/leaderboard/functions.ts

```diff
@@ -65,67 +64,48 @@ export const getLeaderboardData = async (
 export default async function fetchGitHubReleases(
   sliceLimit: number,
 ): Promise<Release[]> {
-  const accessToken = getGitHubAccessToken();
-
-  if (!accessToken) {
-    return [];
-  }
-
-  const response = await fetch("https://api.github.com/graphql", {
-    method: "POST",
-    headers: {
-      Authorization: `Bearer ${accessToken}`,
-      "Content-Type": "application/json",
-    },
-    body: JSON.stringify({
-      query: `{
-        organization(login: "${env.NEXT_PUBLIC_GITHUB_ORG}") {
-          repositories(first: 100) {
-            nodes {
-              name
-              releases(first: 10, orderBy: {field: CREATED_AT, direction: DESC}) {
-                nodes {
-                  name
-                  createdAt
-                  description
-                  url
-                  author {
-                    login
-                    avatarUrl
-                  }
-                  mentions (first: 10) {
-                    nodes {
-                      login 
+  const response: ReleasesResponse = await octokit.graphql({
+    query: `
+        query GetReleases($org: String!) {
+          organization(login: $org) {
+            repositories(first: 100) {
```

Comment:
order by updated at descending

Reply:
![image](https://github.com/coronasafe/leaderboard/assets/99136186/2b4b9b5f-7c6c-4f9a-88d9-ab629a70724c)

It's not supporting UPDATED_AT ,
I think we have to sort the fetched releases based on their updated at field.?

Reply:
No, it is supported. I meant to sort repositories by updated at, not releases. releases can remain created at itself.

Reply:
![image](https://github.com/coronasafe/leaderboard/assets/99136186/b4bf4352-b570-4f91-acb2-9fe18297599c)

Is it fine okk?

Reply:
yup